### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#169] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-30 171427](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/86b8e083-0908-4de3-a134-faa18696b2c2)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione confermata in classe 
Considero una schema di Bernoulli $\Omega=\\{(w_1,...,w_n)|w_i=T,C\\}$ ovvero tutte le possibili parole di $n$ lettere scelte tra $T,C$. 
L'evento che studiamo è $A=\\{ottenere\ a\ T\ prima\ di\ b\ C\\}=\\{parole\ con\ a\ T\ prima\ di\ b\ C\\}$.

Consideriamo le parole lunghe $n=a+b$ composte da $a$ Teste e $b$ Croci, siccome a noi interessano solamente quelle in cui abbiamo $a$ Teste prima delle $b$ Croci possiamo fissare una Croce come ultima lettera e calcolare la probabilità di ottenere $a$ Teste e $b-1$ Croci nelle prime $n-1$ lettere.
$$\mathbb{P}(A)=\binom{a+b-1}{a}p^a(1-p)^{b-1}$$

**Reply by CasuFrost** [reply]
Scusa ma così non staresti considerando solamente la probabilità di ottenere $a$ volte testa in $a+b-1$ lanci ? Non dovresti considerare 
tutti i casi in cui si fanno anche dagli $a$ lanci fino agli $a+b-2$ lanci e si ottiene comunque un numero $a$ di teste? Oppure questi sono già contenuti nel risultato che hai dato?

Presunto risultato alternativo : 
$$\displaystyle\sum_{i=a}^{a+b-1}\binom{i}{a}p^a(1-p)^{i-a}$$
Credo però di non star considerando le intersezioni

**Reply by cornflexxx** [reply]
La soluzione proposta sopra è stata confermata anche dal prof però come fai notare in questo modo stiamo contando solo quei casi in cui ci sono esattamente $b$ croci.  Per calcolare la probabilità in modo corretto:

$$P(\text{a prima di b}) = \sum_{k=a}^{a+b-1} \binom{a+b-1}{k} p^k(1-p)^{a+b-1-k}$$

così consideriamo anche i quei casi in cui  $n° teste>a$.
